packages/cli/package.json

```diff
@@ -7,7 +7,7 @@
     "si": "./src/bin/index.ts"
   },
   "scripts": {
-    "start": "ts-node ./src/index",
+    "start": "ts-node ./src/bin/index",
```

Comment:
why this change? it seems unrelated fo arg parsing.

Reply:
it's needed to run and test this package locally

Reply:
./src/index file does not exists ;)

Reply:
ah, so it was broken from the start? pity.